README.md

```diff
@@ -31,11 +30,25 @@ Load the `dragscroll.js` in a preferable way (that is an UMD module):
 <script src="path/to/dragscroll.js"></script>
 ```
 
-Add the `dragscroll` class to a scrollable element:
+Add a style to `dragscroll` class:
+```css
+.dragscroll{
```

Comment:
I would explain here in details, why users might need every property. Maybe I wish to keep the scrollbar for something?